tools/scripts/scvi/scvi-config.yaml

```diff
@@ -0,0 +1,25 @@
+census:
+  organism: 
+    "mus_musculus"
+  obs_query:
+    'tissue_general == "heart" and is_primary_data == True'
+hvg:
+  top_n_hvg: 
+    8000
+  hvg_batch: 
+    dataset_id
+  min_genes:
+    300
+model:
+  n_hidden: 512
+  n_latent: 200
+  n_layers: 1
+  dropout_rate: 0.1
+train:
+  max_epochs: 10
+  batch_size: 1048
+  plan_kwargs: 
```

Comment:
not needed? or move `training_plan` to here

Reply:
This was a leftover